app/controllers/exports_controller.rb

```diff
@@ -0,0 +1,11 @@
+class ExportsController < ApplicationController
```

Comment:
I figured it out... seemed to be the call to `book` (not sure how that was working...?) rather than referencing `@workbook` in the `SpreadsheetExporter`.